HW12-ATM/src/main/java/homework/BasicATM.java

```diff
@@ -0,0 +1,73 @@
+package homework;
+import java.util.*;
+
+// Основной класс ATM (реализует принципы SOLID, включая открытость-закрытость для модификаций)
+class BasicATM implements ATM {
```

Comment:
Если комментируете класс, то лучше сразу писать javadoc-комментарии.

```suggestion
/**
 * Основной класс ATM (реализует принципы SOLID, включая открытость-закрытость для модификаций)
 */
class BasicATM implements ATM {
```